### DISCUSSION TITLE
Problems with forcibly sampled

### DISCUSSION THREAD

**Discussion by tracyFizz** [apache/skywalking#11428] [Q&A] [answered]
I set the sampling rate of the server to 10%, what way to ensure that a request is forcibly sampled, except to set the sampling rate to 100%.

**Comment by wu-sheng** [comment]
If the downsample is sampled, the agent could ignore the rate and keep tracing.

**Reply by tracyFizz** [reply]
Thank you for your reply. Maybe I didn't make it clear. When I said that a request was forced to be sampled, I meant that the request could be displayed in skywalking UI. That is, when the collector receives the data transmitted by the agent, how can it ensure that the requested link data can be stored?

**Reply by wu-sheng** [reply]
If you mean for the OAP side, no, there is not such a thing. It is hard to follow which use cases make you asking for this.

**Reply by tracyFizz** [reply]
I set the sampling rate of the server to 10%, how to sample 100% of the requests generated by one of the interfaces of the service. Is there such a function?

**Reply by wu-sheng** [reply, marked as answer]
Like I said, no, there isn't. Once you set up sampling for services at the OAP side, there is no other option to override it.

**Reply by tracyFizz** [reply]
ok, thank you
